packages/renderer/src/lib/ui/ProviderResultPage.svelte

```diff
@@ -108,8 +108,8 @@ function onSeverityClicked(severity: 'critical' | 'high' | 'medium' | 'low' | 's
     {@render headerInfo()}
   </div>
   <div class="mb-2 flex flex-row pr-12 pb-2">
```

Comment:
please consider adding region to a most upper div to denote a parent of the provider result POM. role=region, aria-label=image result provider - approximately.